static/app/views/projectInstall/platformOrIntegration.tsx

```diff
@@ -1,28 +1,72 @@
-import * as qs from 'query-string';
+import {useState} from 'react';
+import type {RouteComponentProps} from 'react-router';
 
 import {OnboardingContextProvider} from 'sentry/components/onboarding/onboardingContext';
+import platforms from 'sentry/data/platforms';
+import {t} from 'sentry/locale';
+import type {PlatformIntegration} from 'sentry/types/project';
 import {platformToIntegrationMap} from 'sentry/utils/integrationUtil';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
 
 import GettingStarted from './gettingStarted';
 import {ProjectInstallPlatform} from './platform';
 import PlatformIntegrationSetup from './platformIntegrationSetup';
 
-type Props = React.ComponentProps<typeof ProjectInstallPlatform> &
-  Omit<React.ComponentProps<typeof PlatformIntegrationSetup>, 'integrationSlug'>;
-
-function PlatformOrIntegration(props: Props) {
-  const parsed = qs.parse(window.location.search);
-  const {platform} = props.params;
-  const integrationSlug = platform && platformToIntegrationMap[platform];
-  // check for manual override query param
-  // TODO(priscila): check this case
-  if (integrationSlug && parsed.manual !== '1') {
-    return <PlatformIntegrationSetup integrationSlug={integrationSlug} {...props} />;
+const allPlatforms: PlatformIntegration[] = [
+  ...platforms,
+  {
+    id: 'other',
+    name: t('Other'),
+    link: 'https://docs.sentry.io/platforms/',
+    type: 'language',
+    language: 'other',
+  },
+];
+
+type Props = RouteComponentProps<{projectId: string}, {}>;
+
+function PlatformOrIntegration({params}: Props) {
+  const organization = useOrganization();
+  const [integrationUseManualSetup, setIntegrationUseManualSetup] = useState(false);
```

Comment:
Shouldn't we store this state in the URL, like before?

Reply:
I decided to go with component state, just as we do when users install our JavaScript platform using 'yarn' or 'npm' 

https://github.com/user-attachments/assets/6b86d692-e31b-448e-bab5-a27964158e68

+ Currently we have components state + URL param, so I just removed the URL part. See https://github.com/getsentry/sentry/blob/9ded21a2fcf5db1240b93355823f996b3389349d/static/app/views/onboarding/setupDocs.tsx#L75-L76

Reply:
hmm, k. Personally I would not expect this state to be lost on reload. However in both case it is maybe an advantage as there is no way to go back otherwise... (we should fix that at some point)

Reply:
yes! I would prefer to have the states in the URL too, but if we update this, then we update for both at once